installer/pkg/workflow/destroy.go

```diff
@@ -1,5 +1,28 @@
 package workflow
 
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset"
+)
+
+var (
+	kubeconfigPath = filepath.Join("generated", "auth", "kubeconfig")
+)
+
+const (
+	machineSetNamespace = "default"
```

Comment:
let's not store machines in the default namespace, please move them under something like `openshift-machines`

Reply:
Yeah, this is just what's used in the machine-api-operator right now. Planned on fixing it. I can open a PR there.

Reply:
Fixed. Here and via https://github.com/openshift/machine-api-operator/pull/41.